ooiservices/config/config.yml

```diff
@@ -0,0 +1,5 @@
+TABLEDAP: http://data-test.ooi.rutgers.edu/erddap/tabledap/
+ERDDAP_URL: http://data-test.ooi.rutgers.edu/erddap
+DEBUG: False
+UFRAME_DATA: http://localhost:12570/sensor/m2m/inv
+CACHE_TYPE: simple
```

Comment:
This was originally used for the UFRAME_DATA variable and I was going to use it for the CACHE_TYPE as well.  Straight Copied from front end.